### ISSUE TITLE
Discussions

### ISSUE THREAD

**Issue by defern** [GSA/Crowdsource-Testing-SBA.gov_7-2015#1]
Are you having any issues with anything in the test-cycle or did you find anything that may help other testers?
Let's discuss it!

**Comment by jeremiak** [comment]
Sorry to bring this up at the end of the work week @defern but a quick question: Sometimes the site is "passing" but I might have a suggestion for the way it can be displayed. For instance, there is a display of a bulleted list in some of the event descriptions but they aren't each displayed on an individual line. Would the best thing to do be mark it as "passed" on the survey but open an issue and tag it with enhancement?

Have a good weekend!

**Comment by jpgsa** [comment]
Yes, I would "pass" it on the script and create a new issue in GH when you think there may be a "more powerful way" to do it. :) 

Jacob
